.github/workflows/build_cmake_tarball.yml

```diff
@@ -79,6 +79,9 @@ jobs:
       if: ${{ inputs.LESS_TESTS }}
       run: export LESS_TEST_OPTION="-DT8CODE_ENABLE_LESS_TESTS=ON"
           && echo LESS_TEST_OPTION="$LESS_TEST_OPTION" >> $GITHUB_ENV
+    - name: compiler-optimization
+      if: ${{ inputs.BUILD_TYPE == 'Debug' }}
+      run: export -DCMAKE_C_FLAGS_DEBUG="-O1 -g" -DCMAKE_CXX_FLAGS_DEBUG="-O1 -g"
```

Comment:
Do you maybe want to define a new input which gets passed to `CONFIG_OPTIONS`? Then we can control this in the central CI script. Something like

Input: addtitional_config_options

Reply:
No, because these would be additional options. I want to change the options used when the Debug mode is used, which are written in CMAKE_C/CXX_FLAGS_DEBUG. Additional config options would just append to that.